### PR TITLE
generate kube: handle entrypoint

### PR DESCRIPTION
**Pull request by vrothberg**
The spec of a Kube Container has a `Command` and `Args`.  While both are
slices, the `Command` is the counterpart of the entrypoint of a libpod
container.  Kube is also happily accepting the arguments to as following
items in the slice but it's cleaner to move those to `Args`.

Fixes: #9211
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@haircommander PTAL

File: libpod/kube.go [hunk 1]
```diff
@@ -353,22 +353,21 @@ func containerToV1Container(c *Container) (v1.Container, []v1.Volume, *v1.PodDNS
...
+		kubeContainer.Command = c.Command()
+	}
 
+	kubeContainer.Name = removeUnderscores(c.Name())
 	_, image := c.Image()
 	kubeContainer.Image = image
 	kubeContainer.Stdin = c.Stdin()
 
-	// prepend the entrypoint of the container to command
-	if ep := c.Entrypoint(); len(c.Entrypoint()) > 0 {
```
Comment (mheon) [inline libpod/kube.go:364]: Do we have any tests verifying entrypoint from the image is properly used?

Comment (vrothberg) [reply libpod/kube.go:364]: Good thinking.  I don't think so no.  I'll add another test :+1:

Comment (haircommander) [reply libpod/kube.go:364]: I believe we do for play kube specifically, but I am not sure about generate

Comment (haircommander) [reply libpod/kube.go:364]: stale github state had me answering a question that was already addressed :upside_down_face:

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
Would like a final LGTM from @haircommander before merge

**Comment by haircommander** [comment]
/lgtm

thanks for taking this @vrothberg

**Pull request merged**